core/src/main/resources/jenkins/console/ConsoleUrlProviderGlobalConfiguration/config.properties

```diff
@@ -0,0 +1 @@
+description=Controls what visualization tool handles build console links. Providers higher in the list take precedence over providers lower in the list.
```

Comment:
```suggestion
description=Controls what visualization tool handles build console links. Providers higher in the list take precedence over providers lower in the list. If no explicitly specified provider can render a console link, the default provider will be used.
```

Reply:
Added in [64c3faa](https://github.com/jenkinsci/jenkins/pull/8321/commits/64c3faa8ac16742878377db5f9b0d14ff95a1338), along with some other text.